skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/plugin/interceptor/enhance/ClassEnhancePluginDefine.java

```diff
@@ -70,8 +69,15 @@
             for (ConstructorInterceptPoint constructorInterceptPoint : constructorInterceptPoints) {
                 newClassBuilder = newClassBuilder.constructor(constructorInterceptPoint.getConstructorMatcher())
```

Comment:
@raphw , I have notice your response on https://github.com/raphw/byte-buddy/issues/227. Maybe there is no other way?